src/devices_models/devices/common/pm_translator.jl

```diff
@@ -107,6 +107,30 @@ function get_branch_to_pm(ix::Int, branch::PSY.Line)
     return PM_branch
 end
 
+function get_branch_to_pm(ix::Int, branch::PSY.MonitoredLine)
```

Comment:
No need for this function. You can just make the function in line 86 be able to take `<: ACLine`